dist/clients/Device.js

```diff
@@ -288,20 +288,22 @@ var Device = function (_EventEmitter) {
 
```

Comment:
I rebased last dev and rebuild the project with your changes. so Idk why my babel works not like yours.